make/modules/java.base/gensrc/GensrcCharsetMapping.gmk

```diff
@@ -46,7 +46,15 @@ CHARSET_STANDARD_OS := stdcs-$(OPENJDK_TARGET_OS)
 
 J9TOOLS_DIR := $(SUPPORT_OUTPUTDIR)/j9tools
 JPP_JAR     := $(J9TOOLS_DIR)/jpp.jar
-JPP_TAGS    := PLATFORM-$(OPENJ9_PLATFORM_CODE)
+JPP_TAGS    := PLATFORM-$(OPENJ9_PLATFORM_CODE) OPENJDK_THREAD_SUPPORT
```

Comment:
What is the point of OPENJDK_THREAD_SUPPORT? It doesn't seem to be used.

Reply:
Removed. I was on JDK19 repo.

Reply:
Even for jdk19 I didn't find it used.

Reply:
It is defined at https://github.com/ibmruntimes/openj9-openjdk-jdk19/blob/1fbf0f268122a7b76541ee66afb6f5c5b9c95c9c/closed/GensrcJ9JCL.gmk#L39
Yeah, it seems not used, will open a PR to delete it.